src/test/java/org/kainos/ea/integration/AuthControllerIntegrationTest.java

```diff
@@ -0,0 +1,49 @@
+package org.kainos.ea.integration;
+
+import io.dropwizard.configuration.ResourceConfigurationSourceProvider;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.kainos.ea.FloorIsJavaApplication;
+import org.kainos.ea.FloorIsJavaConfiguration;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.kainos.ea.cli.Login;
+
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.Form;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class AuthControllerIntegrationTest {
+
+    static final DropwizardAppExtension<FloorIsJavaConfiguration> APP = new DropwizardAppExtension<>(
+            FloorIsJavaApplication.class, null,
+            new ResourceConfigurationSourceProvider()
+    );
+    @Test
+    void loginWithValidLogin_ShouldReturnLogin() {
+        Login testLogin = new Login("admin", "admin");
```

Comment:
don't hardcode valid creds, pass in as env vars

Reply:
Updated to use env vars. Also updated README to reflect these changes